src/main/kotlin/com/team4099/robot2020/subsystems/Drive.kt

```diff
@@ -270,23 +270,23 @@ object Drive : Subsystem {
      */
     @Synchronized
     fun setOpenLoop(signal: DriveSignal) {
+        setLeftRightPower(
+            signal.leftMotor * Constants.Drive.MAX_LEFT_OPEN_LOOP_POWER,
+            signal.rightMotor * Constants.Drive.MAX_RIGHT_OPEN_LOOP_POWER
+        )
         if (currentState !== DriveControlState.OPEN_LOOP) {
             leftMasterTalon.configNominalOutputForward(0.0, Constants.Universal.CTRE_CONFIG_TIMEOUT)
             rightMasterTalon.configNominalOutputForward(0.0, Constants.Universal.CTRE_CONFIG_TIMEOUT)
             currentState = DriveControlState.OPEN_LOOP
             HelixEvents.addEvent("DRIVETRAIN", "Entered open loop control")
         }
         brakeMode = if (signal.brakeMode) NeutralMode.Brake else NeutralMode.Coast
-        setLeftRightPower(
-                signal.leftMotor * Constants.Drive.MAX_LEFT_OPEN_LOOP_POWER,
-                signal.rightMotor * Constants.Drive.MAX_RIGHT_OPEN_LOOP_POWER
-        )
     }
 
     @Synchronized
     private fun setLeftRightPower(left: Double, right: Double) {
-        leftMasterTalon.set(ControlMode.PercentOutput, left)
-        rightMasterTalon.set(ControlMode.PercentOutput, right)
+        leftMasterTalon.set(ControlMode.PercentOutput, left, DemandType.ArbitraryFeedForward, 0.0)
+        rightMasterTalon.set(ControlMode.PercentOutput, right, DemandType.ArbitraryFeedForward, 0.0)
```

Comment:
This us more of a workaround than a fix, make sure the setter in lazytalon works instead